src/docfx/lib/markdown/HtmlExtension.cs

```diff
@@ -52,7 +68,7 @@ string ProcessHtml(string html, MarkdownObject block, ErrorBuilder errors)
                 HtmlUtility.TransformLink(ref token, block, getLink!);
```

Comment:
We need a dedicated `HtmlUtility.CountHtmlUsage(string html, Dictionary<(string element, string? attribute)>)` method and run it in line 62 before `HtmlUtility.TransformHtml` to ensure the usage collected is true. E.g., running it inside `SanitizeHtml` would ignore HTML removed by `RemoveRerunCodepenIframes` method.

Also add a `HtmlSantizer.IsAllowed(string element, string? attribute)` method to determine if an element is allowed.